test/api/requests/site.test.js

```diff
@@ -558,35 +563,39 @@ describe('Site API', () => {
         .catch(done);
     });
 
-    it('should respond with a 400 if the site already exists', (done) => {
-      const userPromise = factory.user();
+    it('should respond with a 400 if the site already exists', async () => {
+      const user = await factory.user();
```

Comment:
If you want to simplify some of the setup logic, there is a test utility, `createSiteUserOrg` that might help here

Reply:
Fixed